### PR TITLE
feat(gatsby-plugin-image): Add helper function to grab src

### PR DESCRIPTION
**Pull request by laurieontech**
The path to get the src of an image is quite long. Adding a helper function makes this a better DX.

### CODE REVIEW

File: packages/gatsby-plugin-image/README.md [hunk 1]
```diff
@@ -224,6 +224,8 @@ export const query = graphql`
 `
 ```
 
+If you need the image `src` directly you can use the `getSrc` helper function, which is equivalent to `data.file.childImageSharp.gatsbyImageData.images.fallback.src`.
```
Comment (laurieontech) [inline packages/gatsby-plugin-image/README.md:227]:
```suggestion
If you need the image `src` directly you can import the `getSrc` helper function from `gatsby-plugin-image`. That function is equivalent to `data.file.childImageSharp.gatsbyImageData.images.fallback.src`.
```

**Comment by ascorbic** [comment]
Might be worth noting (or changing) the fact that if the only format is webp, there is no src just a srcset. As there is no browser that supports webp but not picture elements, we don't bother with the fallback for them.

**Review by ascorbic — DISMISSED** [review]
Looks great

**Comment by laurieontech** [comment]
Hmm, is there a scenario where we only produce webp? I thought that was always extra.

**Comment by ascorbic** [comment]
If they specify `formats=[WEBP]`, or if the input file is webp

**Pull request merged**